DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule/Mvc/Controllers/BaseJobProfileWidgetController.cs

```diff
@@ -49,30 +50,102 @@ protected JobProfile CurrentJobProfile
 
         public string GetDynamicTitle(bool skipNoTitle)
         {
+            var changedTitle = CheckForAcronym(CurrentJobProfile.Title);
             switch (CurrentJobProfile.DynamicTitlePrefix)
             {
                 case "No Prefix":
-                    return $"{CurrentJobProfile.Title}";
+                    return $"{changedTitle}";
 
                 case "Prefix with a":
-                    return $"a {CurrentJobProfile.Title}";
+                    return $"a {changedTitle}";
 
                 case "Prefix with an":
-                    return $"an {CurrentJobProfile.Title}";
+                    return $"an {changedTitle}";
 
                 case "No Title":
-                    return skipNoTitle ? GetDefaultDynamicTitle(CurrentJobProfile.Title) : string.Empty;
+                    return skipNoTitle ? GetDefaultDynamicTitle(changedTitle) : string.Empty;
 
                 default:
-                    return GetDefaultDynamicTitle(CurrentJobProfile.Title);
+                    return GetDefaultDynamicTitle(changedTitle);
             }
         }
 
-        /// <summary>
-        /// Indexes this instance.
-        /// </summary>
-        /// <returns>Redirect</returns>
-        public ActionResult BaseIndex()
+        public string CheckForAcronym(string title)
+        {
+            var splitTitle = title.Split(' ');
+
+            var newTitle = string.Empty;
+
+            foreach (var word in splitTitle)
+            {
+                string newWord = ChangeWordCase(word);
+                newTitle = $"{newTitle} {newWord}";
+            }
+
+            return newTitle.TrimStart();
+        }
+
+        public bool SpecialConditionWords(string word)
+        {
+            var specialConditionWords = new[]
```

Comment:
Need the comment that points to the new story in the backlog to make these special titles amendable in CMS